src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java

```diff
@@ -99,9 +99,31 @@ public void ChatTestsStaff_003_sendPrivateMessage() {
 
   @DontRun(env = {"prod"})
   @Test(groups = {"ChatTestsStaff_004", "ChatStaff", "ChatTests"})
-  public void ChatTestsStaff_004_basicUserChatFails() {
+  public void regularUserCanNotEnterChatOnPreviewEnvironment() {
     openChatForUser(credentials.userName10, credentials.password10);
     SpecialPageObject special = new SpecialPageObject();
     special.verifyPageHeader("Permissions error");
   }
+
+  @DontRun(env = {"sandbox"})
```

Comment:
what is the reason of blocking ```ChatTests``` from running on dev, preview and sandbox environments and ```ChatTestsStaff``` only on sandbox?